miden-lib/asm/miden/kernels/tx/account.masm

```diff
@@ -355,6 +360,31 @@ export.set_item
     # => [V]
 end
 
+#! Sets an item in the specified account storage map. Panics if no map is found for ROOT.
+#!
+#! Stack: [index, KEY, VALUE, ROOT, ...]
+#! Output: [OLD_VALUE, NEW_ROOT, ...]
+#!
+#! - ROOT is the root of the map to set the KEY NEW_VALUE pair.
+#! - NEW_VALUE is the value to set under KEY.
+#! - KEY is the key to set.
+#! - OLD_VALUE is the previous value of the item.
+#! - NEW_ROOT is the new root of the map.
```

Comment:
Missing docs for `index`.
The `Stack` should probably be renamed to `NEW_VALUE` too
It would be nice to rename `ROOT` to `OLD_ROOT` to keep things consistent with value

Reply:
good point